recipes-bsp/sgx/sgx-dcap_1.9.bb

```diff
@@ -0,0 +1,105 @@
+# Packages supported
+# sgx-dcap : SGX Data Center Attestation Package.
+
+### common ###
+
+SUMMARY = "Intel(R) SGX Data Center Attestation Primitives (DCAP) driver"
+LICENSE = "BSD & Intel-Redistributable-Binaries"
+LIC_FILES_CHKSUM = "file://License.txt;md5=66da782d3c2c41b9669340fb2704ac59"
+
+# Packages required to build.
+DEPENDS += "curl sgx-sdk-cross"
+
+# Source locations
+S = "${WORKDIR}/git"
+D_prebuilt_dcap = "${S}/QuoteGeneration"
+D_sgxssl="${S}/QuoteVerification/sgxssl"
+D_openssl="${S}/QuoteVerification/sgxssl/openssl_source"
+
+SRC_URI  = "git://git@github.com/intel/SGXDataCenterAttestationPrimitives.git"
```

Comment:
Do not leave git@ username in here, add ;protocol=https.

Reply:
done